src/single-spa-react.test.js

```diff
@@ -55,7 +55,7 @@ describe("single-spa-react", () => {
   });
 
   it(`mounts and unmounts a React component, passing through the single-spa props`, () => {
-    const props = { why: "hello", customProps: {} };
```

Comment:
The tests were originally written before the change to customProps that I described in the description of this PR